libs/neo4j/langchain_neo4j/vectorstores/neo4j_vector.py

```diff
@@ -921,6 +923,8 @@ def from_existing_index(
         search_type: SearchType = DEFAULT_SEARCH_TYPE,
         keyword_index_name: Optional[str] = None,
         embedding_dimension: Optional[int] = None,
+        text_node_properties: Optional[List[str]] = None,
+        retrieval_query: Optional[str] = None,
```

Comment:
```suggestion
        retrieval_query: str = "",
```
Keeps it consistent with `from_existing_graph`